Dallia_Sintique/sistema_notas.py

```diff
@@ -0,0 +1,177 @@
+from dataset_alunas import dataset
+
+def obter_opcao():
+    codigo_opcao = 0
+
+    while codigo_opcao not in [1, 2, 3, 4, 5, 6]:
+        try:
+            codigo_opcao = int(input("\nEscolha uma opção:\n"
+                                    "1 - Incluir uma nova aluna\n"
+                                    "2 - Consultar lista de alunas\n"
+                                    "3 - Consultar faltas da aluna\n"
+                                    "4 - Consultar notas da aluna\n"
+                                    "5 - Consultar status de aprovação\n"
+                                    "6 - Sair do sistema\n"
+                                    "Opção: "))
+
+            if codigo_opcao not in [1, 2, 3, 4, 5, 6]:
+                print("Opção inválida. Por favor, escolha uma opção válida (1 a 5).\n")
+        except ValueError:
+            print("Entrada inválida. Por favor, digite um número inteiro.\n")
+
+        return codigo_opcao
+
+def obtem_nome(pergunta):
+  while True:
+    entrada_usuário = str(input(pergunta)).lstrip().rstrip()
+    if entrada_usuário.replace(" ", "").isalpha() and entrada_usuário != "":
+      break
+    else:
+      print("Digite um nome válido")
+  return entrada_usuário
+
+def obtem_nota(numero_nota, tipo):
+  status = 0
+  while status == 0:
+    try:
+        if tipo == 'n':
+            nota = float(input(f"Digite a nota {numero_nota} da aluna) (0 a 10): "))
+        else:
+            nota = float(input(f"Digite a nota de participação da aluna) (0 a 10): "))
+        if nota >= 0 and nota <= 10:
+            status = 1
+        else:
+            print("Digite uma nota válida entre 0 e 10")
+    except:
+      print(" Entrada inválida. Digite um número inteiro entre 0 e 10")
+  return nota
+
+def obtem_presenca(num_presenca):
+  contador = 0
+  while contador == 0:
+    presenca = input(f"Digite a presença {num_presenca} da aluna apenas com (True ou False): ").strip()
+    if presenca.lower() in ['true', 'false']:
+      presenca = presenca.lower() == 'true'
+      contador = 1
```

Comment:
Podemos dar return neste ponto, sem precisar atribuir a uma variável

**Sugestão:**
```suggestion
      return presenca.lower() == 'true'
```